.github/workflows/ci.yml

```diff
@@ -83,6 +83,11 @@ jobs:
             command: bin/test.sh 'sbt-metals/scripted; slow/testOnly -- tests.sbt.*'
             name: Sbt integration
             os: ubuntu-latest
+          - type: sbt-metals jdk17
+            command: bin/test.sh sbt-metals/scripted
```

Comment:
Are our current tests covering this issue?

Reply:
Ouch. Thx for pointing at this.
Now it checks